test/test_plantuml.py

```diff
@@ -0,0 +1,22 @@
+from test_simple_on_off import sm
```

Comment:
I'm not sure the generated staruml covers all capabilities of the library. Ie. Can it generate a multi level HSM? What about historical states? Can you add tests to cover these scenarios?